### ISSUE TITLE
pytorch 1.12.0 binary wheels please

### ISSUE THREAD

**Issue by stas00** [rusty1s/pytorch_scatter#306]
as with every new pt release - could we please have 1.12 binaries please and thank you!

**Comment by rusty1s** [comment]
Working on it. Please not that you may also try to switch to `torch.scatter_reduce` in the new PyTorch release.

**Comment by stas00** [comment]
indeed, it's the plan, but I thought it'd be good to have a back up - hopefully it just works and we won't bother you again with every new pytorch release.

**Comment by stas00** [comment]
thank you very much, @rusty1s, for the quick release!

I am asking our testing team to validate that all is good.

**Comment by NielsRogge** [comment]
Hi @rusty1s, thanks for the quick response.

I tried to replace `scatter` from this library by `tensor.scatter_reduce`. The latter expects an output tensor, so what I do to make it equivalent is the following:

```
from torch_scatter import scatter

src = torch.tensor([5,1,7,2,3,2,1,3])
index = torch.tensor([0,0,1,0,2,2,3,3])

out = scatter(src, index, dim=0, reduce="max")
print(out)
```

vs. PyTorch:
```
import torch

src = torch.tensor([5,1,7,2,3,2,1,3])
index = torch.tensor([0,0,1,0,2,2,3,3])

out = torch.zeros((index.max().item()+1), dtype=src.dtype)
out = out.scatter_reduce(dim=0, index=index, src=src, reduce="amax", include_self=False)
print(out)
```
This is giving me equivalent results for the forward pass, but fails on the backward pass with 

```shell
RuntimeError: one of the variables needed for gradient computation has been modified by an inplace operation: [torch.FloatTensor [416]], which is output 0 of ScatterReduceBackward0, is at version 3; expected version 0 instead. Hint: enable anomaly detection to find the operation that failed to compute its gradient, with torch.autograd.set_detect_anomaly(True).
```

Posted a thread on PyTorch's forum here: https://discuss.pytorch.org/t/runtimeerror-function-powbackward0-returned-nan-values-in-its-0th-output-when-using-scatter-reduce/155307.

If you have any clues on how to properly replace scatter with native PyTorch, any help is appreciated.

**Comment by rusty1s** [comment]
Do you have a reproducible example for the backward pass?

**Comment by NielsRogge** [comment]
Here are the steps to reproduce it:

1) clone Transformers:
```
git clone https://github.com/huggingface/transformers
cd transformers
pip install -e .
```
You might have to `pip install pytest` as well.
2)  Run the TAPAS tests (TAPAS is the model that uses `scatter`). From the root of the repo:
```
RUN_SLOW=yes pytest tests/models/tapas/test_modeling_tapas.py::TapasModelTest::test_training
```

**Comment by rusty1s** [comment]
Thanks. Any chance you have a more minimal example? :)
